### PR TITLE
added docs and small owner fix

### PR DESCRIPTION
**Pull request by mihaicalinluca**

### CODE REVIEW

File: contracts/mvx-game-sc/README.md [hunk 1]
```diff
@@ -1,3 +1,78 @@
...
+- `enable/disable` the contract for maintenance
+- set the `game starting fee` amount
+- set the `token id` for the currency of the SC (used for game starting fee, wager and reward)
+- `send rewards` or return the wager to the users who participated in a specific game
+
+Each player has to pay the `wager` amount set by the game creator in the `token id` set by the owner in order to join the game.
+
+The game is considered `invalid` until the `minimum number of players` have joined the game and if the `waiting time` has passed, no more players can join.
+
+The SC does not have any logic for calculating the winner, so it expects input from the owner with the winnes' addresses and the percentage (*100) of the total reward (sum of wagers) won by each.
```
Comment (psorinionut) [inline contracts/mvx-game-sc/README.md:25]: it expects input from the owner with the winners' addresses

File: contracts/mvx-game-sc/README.md [hunk 2]
```diff
@@ -1,3 +1,78 @@
...
+Each player has to pay the `wager` amount set by the game creator in the `token id` set by the owner in order to join the game.
+
+The game is considered `invalid` until the `minimum number of players` have joined the game and if the `waiting time` has passed, no more players can join.
+
+The SC does not have any logic for calculating the winner, so it expects input from the owner with the winnes' addresses and the percentage (*100) of the total reward (sum of wagers) won by each. 
+
+**The game**:
+- If the game is `invalid`, the `wager` amount will be returned to the players that have joined the game and the `game starting fee` will be returned to the creator
+- If the game is `valid`, but no winners are sent by the owner, such in the case of a tie/draw, the contract will send back the `wager` amount paid by every player who joined
+- If the game is `valid` and winners are sent by the owner, the SC will send the reward to the winners as explained by the owner
```
Comment (psorinionut) [inline contracts/mvx-game-sc/README.md:30]: Maybe rephrase a little, as the owner does not explain anything.
Something like
If the game is `valid` and the winners were provided, the SC will send the rewards to them, based on the input of the owner.

File: contracts/mvx-game-sc/README.md [hunk 3]
```diff
@@ -1,3 +1,78 @@
...
+
+## Endpoints
+### createGame
+```
```
Comment (psorinionut) [inline contracts/mvx-game-sc/README.md:34]: It is better formated if you write the programming language, so add rust after the first ```, like so:
```rust
...
```

File: contracts/mvx-game-sc/README.md [hunk 4]
```diff
@@ -22,16 +22,16 @@ Each player has to pay the `wager` amount set by the game creator in the `token
...
 
-The SC does not have any logic for calculating the winner, so it expects input from the owner with the winnes' addresses and the percentage (*100) of the total reward (sum of wagers) won by each. 
+The SC does not have any logic for calculating the winner, so it expects input from the owner with the winners' addresses and the percentage (*100) of the total reward (sum of wagers) won by each. 
 
 **The game**:
 - If the game is `invalid`, the `wager` amount will be returned to the players that have joined the game and the `game starting fee` will be returned to the creator
-- If the game is `valid`, but no winners are sent by the owner, such in the case of a tie/draw, the contract will send back the `wager` amount paid by every player who joined
-- If the game is `valid` and winners are sent by the owner, the SC will send the reward to the winners as explained by the owner
+- If the game is `valid`, but no winners are provided, such in the case of a tie/draw, the contract will send back the `wager` amount paid by every player who joined
+- If the game is `valid` and winners are provided, the SC will send the reward to the them, based on the input of the owner.
```
Comment (psorinionut) [inline contracts/mvx-game-sc/README.md:30]: send the rewards to them

**Review by psorinionut — APPROVED** [review]

**Review by dorin-iancu — APPROVED** [review]

**Review by alyn509 — APPROVED** [review]
